src/pages/projects/form/RelatedProjectsField.tsx

```diff
@@ -0,0 +1,88 @@
+import { Autocomplete, Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import { useState } from 'react';
+import { Field, FieldProps } from 'formik';
+import { useTranslation } from 'react-i18next';
+import { CristinApiPath } from '../../../api/apiPaths';
+import { ProjectChip } from '../../registration/description_tab/projects_field/ProjectChip';
+import { SearchResponse } from '../../../types/common.types';
+import { CristinProject } from '../../../types/project.types';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { useDebounce } from '../../../utils/hooks/useDebounce';
+import { useFetch } from '../../../utils/hooks/useFetch';
+import { getLanguageString } from '../../../utils/translation-helpers';
+import { AutocompleteTextField } from '../../../components/AutocompleteTextField';
+import { EmphasizeSubstring } from '../../../components/EmphasizeSubstring';
+import { ProjectFieldName } from './ProjectFormDialog';
+
+export const RelatedProjectsField = () => {
```

Comment:
Veldig mye likhet med denne komponenten og `ProjectsField` i Registration-wizard. Prøvde å dra ut i en fellesting først, men ble ikke så rett frem som jeg håpte pga at den også må vise valgt prosjekt og de har ulike modeller i de to tilfellene :/